### DISCUSSION TITLE
代码块中如何使用bat

### DISCUSSION THREAD

**Discussion by fuxingjun** [vuepress/core#966] [Q&A]
现在代码块能支持bat语法吗?

**Comment by Lemmingh** [comment]
取决于你使用的 syntax highlighter。

如果使用 Prism (`@vuepress/plugin-prismjs`)，比如通过 default theme，则支持的 language identifier 是 [`batch`](<https://prismjs.com/#supported-languages>)。

如果使用 Shiki (`@vuepress/plugin-shiki`)，则可用 [`batch` 或 `bat`](<https://github.com/shikijs/shiki/blob/main/docs/languages.md>)。
